src/ApiService/ApiService/onefuzzlib/ProxyOperations.cs

```diff
@@ -54,7 +54,11 @@ public ProxyOperations(ILogTracer log, IOnefuzzContext context)
         _logTracer.Info($"creating proxy: region:{region}");
         var newProxy = new Proxy(region, Guid.NewGuid(), DateTimeOffset.UtcNow, VmState.Init, Auth.BuildAuth(), null, null, _context.ServiceConfiguration.OneFuzzVersion, null, false);
 
-        await Replace(newProxy);
+        var r = await Replace(newProxy);
+        if (!r.IsOk) {
+            _logTracer.Error($"failed to save new proxy {newProxy.ProxyId} due to {r.ErrorV}");
```

Comment:
we should probably error-out here, but maybe we will address this in future, and keep this the same as the Python version for now?

Reply:
absolutely